src/Error/Debugger.php

```diff
@@ -668,16 +668,19 @@ public static function exportVarAsNodes($var, int $maxDepth = 3): NodeInterface
     protected static function export($var, DebugContext $context): NodeInterface
     {
         $type = static::getType($var);
+
+        if (preg_match('/^resource /', $type)) {
```

Comment:
```suggestion
        if (str_starts_with($type, 'resource ')) {
```
PHP 8

Reply:
Thanks, will take a while for it to get the new into the daily vocabulary :)